Tools/ardupilotwaf/boards.py

```diff
@@ -276,6 +277,11 @@ class bebop(linux):
 
     def configure_env(self, cfg, env):
         super(bebop, self).configure_env(cfg, env)
+        os.environ["PKG_CONFIG_PATH"] = '/usr/lib/arm-linux-gnueabihf/pkgconfig'
+        libiio = cfg.check_cfg(package='libiio', mandatory=False)
+        if libiio:
+            cfg.define('LIBIIO', True)
+            env.LIB += ['iio']
```

Comment:
I think the use of _uselib_store='IIO'_ in check_cfg would be preferred. If the test is successful then _HAVE_IIO_ will be defined. The if would probably not be needed - I'm not 100% sure about the LIB part.